app/src/utilities/kinematics_mg400.py

```diff
@@ -54,14 +75,14 @@ def forward_kinematics(angles):
         rot_y(LINK3, j_3) + LINK4
 
     p_x, p_y, p_z = rot_z(pos, j_1)
-    Rz = j_1 + j_4
-    tool_vec = np.round([p_x, p_y, p_z, 0, 0, Rz], decimals=ROUND_DECIMALS)
+    Rx = j_1 + j_4
+    tool_vec = np.round([p_x, p_y, p_z, Rx, 0, 0], decimals=ROUND_DECIMALS)
     return tool_vec
 
 
 def inverse_kinematics(tool_vec):
     """inverse_kinematics"""
-    p_x, p_y, p_z, _, _, Rz = tool_vec
+    p_x, p_y, p_z, Rx, _, _ = tool_vec
```

Comment:
Thank you so much.
This is the issue that we want to fix.

I also work around to fix ROS2 interface
https://github.com/HarvestX/MG400_ROS2/pull/102

After finishing implementation and confirmation, I would merge it.